langchain4j/src/main/java/dev/langchain4j/classification/EmbeddingModelTextClassifier.java

```diff
@@ -98,11 +98,11 @@ public EmbeddingModelTextClassifier(EmbeddingModel embeddingModel,
 
         this.exampleEmbeddingsByLabel = new HashMap<>();
         examplesByLabel.forEach((label, examples) ->
-            exampleEmbeddingsByLabel.put(label, embeddingModel.embedAll(
-                examples.stream()
-                    .map(TextSegment::from)
-                    .collect(toList())).content()
-            )
+                exampleEmbeddingsByLabel.put(label, embeddingModel.embedAll(
```

Comment:
is this formatting according to the new config?

Reply:
should be. Let me update and re-format